lua/treesitter-context.lua

```diff
@@ -486,6 +486,24 @@ local function build_lno_str(lnum, width)
   return string.format('%'..width..'d', lnum)
 end
 
+local function get_relative_line_num(ctx_node_line_num)
+  local cursor_line_num = vim.fn.line(".")
+  local num_folded_lines = 0
+  -- Find all folds between the context node and the cursor
+  local current_line = ctx_node_line_num
```

Comment:
```suggestion
  local current_lnum = ctx_node_line_num
```

The general convention I use is:
- `lnum` is for 1-indexed line numbers
- `row` is for 0-indexed line numbers
- `line` for text of a line